w1r3/go/main.go

```diff
@@ -514,6 +583,20 @@ func histogramBoundaries() []float64 {
 	return boundaries
 }
 
+func cpuHistogramBoundaries() []float64 {
+	boundaries := make([]float64, 0)
+	boundary := 0.0
+	increment := 0.1 / float64(time.Second)
```

Comment:
If we are doing per-byte this seems too large?

Reply:
AFAICT  `time.Second` is $10^9$ [^1], so `increment` is initially $10^{-10}s == 0.1ns$.  Considering we get values in the "< 5ns/byte" range, do we really need more precision?  Do we really believe an improvement of $0.01ns$ is real?

And remember: we only have 200 buckets. If we set them at $0.01$ we would need to skip the first nano to get good resolution around the expected values (and the expected values vary by language).

With that said, I am happy to change the histogram in any way, subject to the constraints of:

1) We only get 200 buckets.
2) I would like these to work for C++, Golang, Java, and Python.

[^1]: https://pkg.go.dev/time#pkg-constants

Reply:
Ahh sorry, I totally misread the code and thought it was 0.1 seconds. Never mind.